docs/reference/faq.md

```diff
@@ -0,0 +1,160 @@
+
+See [Maintainers](../concepts/governance/maintainers.md)
+
+See [Governance Policy](../concepts/governance/policy.md)
+
+## What is the governing model for PyScript?
+
+See [Governance Policy](../concepts/governance/policy.md)
+
+## What is the license?
+
+PyScript uses the Apache-2.0 license.
+
+Pyodide uses MPL-2.0 license. Various packages are distributed under their corresponding license.
+
+## Is Pyodide a replacement for CPython? How does PyScript and Pyodide compare to CPython on WASM?
+
+No. They have different elements that do different things, both of which are additive.
+
+PyScript sits on top of everything. Pyodide came before the work on CPython and WASM. Patches were created in order for it to work, but now that CPython/WASM are progressing, Pyodide is able to remove a few of those patches. Additionally, CPython doesn’t deal with building Python packages for WASM. WASM related work on upstream CPython will integrate into Pyodide in the near future.
+
+For a list of differences from “standard” CPython, see [Pyodide Python compatibility — Version 0.20.0](https://pyodide.org/en/stable/usage/wasm-constraints.html)
+
+## Hasn’t this already been done before by Brython/skulpt?
+
+No. Brython and Skulpt accomplish different things than PyScript and Pyodide.
+
+Brython is client-side and functions as syntax on top of Javascript—it is a reimplementation of Python on top of Javascript, without support for packages or a file system. The extraction of a package in normal Python has been replaced completely by something else. You should be able to run code with minimal changes; however, that isn’t possible with Brython.
+
+Skulpt is a cross compiler from Python to Javascript, leveraged for compatibility with the Python ecosystem. If you want to do any more Python, you would have to send it over.
+
+Fairly similar syntax to normal Python, but not exactly the same. If Brython was an interpreter and a full Python implementation, it could be used with PyScript to leverage packages like Numpy, but that isn’t possible as it stands today. They do have the Python script tag, but it is a smaller API and not as feature rich—which is why PyScript is built on Pyodide, Emscripten, and WebAssembly.
+
+## How can I contribute/help?
+
+**PyScript** - we are currently working on building documentation and a contributing guide. In the meantime, just ask to help on the PyScript [discussions page](https://anaconda.cloud/s/pyscript) or in the [repo](http://github.com/pyscript/pyscript).
+
+**Pyodide** - refer to [Pyodide docs](https://pyodide.org/en/stable/development/contributing.html)
+
+## WebAssembly Security
+
+See [WebAssembly docs](https://webassembly.org/docs/security/#:~:text=The%20security%20model%20of%20WebAssembly,the%20constraints%20of%20(1).)
```

Comment:
Is that period at the end in the right place? Is it part of the URL? If not, it should be on the outside of the parenthesis.

Reply:
I'll add a period at the end.

@fpliger as it stands, this link highlights the first paragraph of that webassembly security page. Is that the intention, or can I just link them to https://webassembly.org/docs/security/   ?

Reply:
I think we can just do https://webassembly.org/docs/security/